packages/netlify-cms-core/src/components/Collection/Entries/EntryCard.js

```diff
@@ -89,9 +90,16 @@ const EntryCard = ({
   viewStyle = VIEW_STYLE_LIST,
 }) => {
   const label = entry.get('label');
-  const title = label || entry.getIn(['data', inferedFields.titleField]);
+  const entryData = entry.get('data');
+  let title = label || entryData.get(inferedFields.titleField);
   const path = `/collections/${collection.get('name')}/entries/${entry.get('slug')}`;
-  let image = entry.getIn(['data', inferedFields.imageField]);
+
+  const summary = collection.get('summary');
+  if (summary) {
+    title = compileSlug(summary, new Date(), getIdentifier(entryData, collection), entryData);
```

Comment:
The date replacements should be based off of fields in the entry, similar to what we do when creating a deploy preview URL: https://github.com/netlify/netlify-cms/blob/3cac3ef7796e89f8a3462b2dc06457fb4086cb29/packages/netlify-cms-core/src/backend.js#L242

Reply:
A problem could arise here if we do not have a date field in the entry.  
We would like to use the `compileSlug` function and "ignore" all date references in `summary` (which `compileSlug` currently doesn't allow, so we'd have to refactor it) or we could catch the raised exception and retry `compileSlug` after removing all date template string from `summary`